EZP-30837: Content forms package (#872)
ezsystems/developer-documentation: docs/releases/ez_platform_v3.0_deprecations.md

## repository-forms

Forms located in `repository-forms` have been moved to other packages.

Content Type editing, including Action Dispatchers, Form Processors, Types and Data classes related to Content Types/Limitations,
has been moved to `ezplatform-admin-ui`.

The following locations have been changed:

| Former location | New location |
|----------|-----|
|`EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface`|`EzSystems\EzPlatformAdminUi\FieldType\FieldDefinitionFormMapperInterface`|
|`EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface`|`EzSystems\EzPlatformAdminUi\Limitation\LimitationFormMapperInterface`|
|`EzSystems\RepositoryForms\Limitation\LimitationValueMapperInterface`|`EzSystems\EzPlatformAdminUi\Limitation\LimitationValueMapperInterface`|

> Forms for content creation have been moved to a new package `ezplatform-content-forms`.

Reviewer: ```suggestion
Forms for content creation have been moved to a new `ezplatform-content-forms` package.
```

> `repository-forms` still remains as an additional layer, do your custom implementations using the package will still work.

Reviewer: ```suggestion
`repository-forms` remains as an additional layer ensuring that your custom implementations that use the package will still work.
```
